README.md

```diff
@@ -1,7 +1,7 @@
 # LaserWeb4-Binaries
 Installers for LaserWeb4
 
-Linux Users: Download from http://laserweb-release.s3-website-us-east-1.amazonaws.com/
+Linux Users: not available, see https://github.com/LaserWeb/LaserWeb4-Binaries/issues/13 and https://github.com/LaserWeb/LaserWeb4-Binaries/issues/12 for details
```

Comment:
Linux Users Look at: https://github.com/LaserWeb/LaserWeb4-Binaries/releases/download/untagged-4818330b6baa8213d4a7/laserweb-builder-v4.0.996-130-x86_64.AppImage

Not up-to-date but working